core/server/types.go

```diff
@@ -1,18 +1,19 @@
-package serverv2
+package server
 
 import (
-	"github.com/spf13/viper"
-
 	coreapp "cosmossdk.io/core/app"
+	"cosmossdk.io/core/log"
 	"cosmossdk.io/core/transaction"
-	"cosmossdk.io/log"
-	"cosmossdk.io/server/v2/appmanager"
 )
 
-type AppCreator[AppT AppI[T], T transaction.Tx] func(log.Logger, *viper.Viper) AppT
+type AppOptions interface {
+	Get(string) interface{}
+}
+
+type AppCreator[AppT AppI[T], T transaction.Tx] func(log.Logger, AppOptions) AppT
 
 type AppI[T transaction.Tx] interface {
-	GetAppManager() *appmanager.AppManager[T]
+	GetAppManager() any
```

Comment:
personally I hate this

Reply:
wanna bring prunes things back to server/v2 so no more of this stuff